assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -150,6 +150,14 @@ table.wc-block-cart-items {
 	display: flex;
 }
 
+.is-small {
```

Comment:
I wonder why this is hidden on mobile?

Reply:
This looks like it was hidden on mobile because of #1587 - I think it looks too crowded on mobile if it's added back, but on medium it looks OK. @mikejolley since you authored the original PR there, what do you think about adding the sale badge back on medium?